src/functions/menus/getNavigationMenu.js

```diff
@@ -0,0 +1,21 @@
+export default function getNavigationMenu(data) {
+  const socialMedia = data[data?.length - 1];
+  const { children } = socialMedia;
+  const menuProps = data.slice(0, 4).map(({ label, path }) => {
+    return {
+      label: label.toUpperCase(),
+      href: path,
+    };
+  });
+  const socialLinks = children.map(({ label, title, url }) => {
+    return {
+      url: title,
+      src: url,
+      label: label.toLowerCase(),
```

Comment:
What's happening here?

Reply:
@kilemensi Not sure what you mean....but we are organizing the social link data in a way we can feed it into the socialMediaIcon as such 
``` items :{
url:"",
src:"",
label:""
}

```
We are also converting the label to lower case for consistency

Reply:
I mean why does title become url, url became src, etc... the lowercasing part is alright @BrendaBucha

Reply:
Ahhhh...okay, so on WordPress we don't necessarily have labels that support the content we are feeding into the menu sections. The menu content block looks like this 👇🏽 

<img width="595" alt="Screenshot 2021-08-17 at 16 54 52" src="https://user-images.githubusercontent.com/89015352/129739041-47d01680-8178-4c25-8be0-8ede7488b06a.png">


Changing the `Navigation label` on wordpress  in the form changes this space here...
<img width="488" alt="Screenshot 2021-08-17 at 16 59 01" src="https://user-images.githubusercontent.com/89015352/129739548-43cb8751-9562-4da9-82bb-be1196a1f952.png">

so the only other inputs we can work with to accomodate the url image and href props required for the SocialMediaIcon is the `Title Attribute` and the` URL`

Reply:
Cool @BrendaBucha :

1. Do we really expect a Twitter icon to change? Otherwise why are we pulling them from CMS rather than just adding icons to `assets` folder and importing them in code?
2. Say we do expect the icons to change (doubt it), wouldn't it be better for the icon to be under Title Attribute i.e. something related to the title and URL to be actual URL to Twitter account?

Reply:
@kilemensi  Um, I think the icons will not change `for now`...but the links will. If you are okay with having to change that in the codebase or having to follow up on content after development  i'll remove social links from the cms

For the naming, I'll update that

Reply:
Lets add a story in PT to revisit this

Reply:
Okay